conf/test.config

```diff
@@ -23,15 +23,16 @@ params {
     input  = params.pipelines_testdata_base_path + 'detaxizer/samplesheets/samplesheet.csv'
 
     // Genome references
-    fasta               = params.pipelines_testdata_base_path + 'detaxizer/host_reference/genome.hg38.chr21_10000bp_region.fa'
+    fasta_bbduk                = params.pipelines_testdata_base_path + 'detaxizer/host_reference/genome.hg38.chr21_10000bp_region.fa'
+
+    // Run bbduk and kraken2
+    classification_bbduk       = true
+    classification_kraken2     = true
```

Comment:
So each of them need to be actively chosen? What would happen by default? No classification at all?